cmd/crank/beta/xpkg/init.go

```diff
@@ -116,14 +138,128 @@ func (c *initCmd) Run(k *kong.Context, logger logging.Logger) error {
 		return errors.Wrapf(err, "failed to get repository's HEAD from %q", repoURL)
 	}
 
-	// TODO(phisco): replace placeholders for the name all around the
-	// 	repository? Maybe we can just agree on some markdown text in the
-	// 	repos to print to let the user know what to do next?
-
 	_, err = fmt.Fprintf(k.Stdout, "Initialized package %q in directory %q from %s (%s)\n", c.Name, c.Directory, getPrettyURL(logger, repoURL, ref), ref.Name().Short())
+	if err != nil {
+		return errors.Wrap(err, "failed to write to stdout")
+	}
+
+	err = c.handleNotes(k.Stdout, logger)
+	if err != nil {
+		return errors.Wrap(err, "failed to handle NOTES.txt")
+	}
```

Comment:
```suggestion
	if err := c.handleNotes(k.Stdout, logger); err != nil {
		return errors.Wrap(err, "failed to handle NOTES.txt")
	}
```

Nit: In a few places you could limit the error scope.

https://github.com/crossplane/crossplane/tree/master/contributing#scope-errors